config/locales/en.yml

```diff
@@ -606,3 +608,6 @@ en:
     gbp: £
     eur: €
     usd: $
+  time:
+    formats:
+      date: '%a, %d %b %Y'
```

Comment:
This seems to be a date format under neath time, and the date and time formats are already defined in lines 2-7 of this file.

Reply:
Poor merging on my part - thank you. I've corrected that.